src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -695,7 +695,7 @@ namespace ExhaustAirSystemManager {
         Real64 Tin = state.dataZoneTempPredictorCorrector->zoneHeatBalance(thisExhCtrl.ZoneNum).ZT;
         Real64 thisExhCtrlAvailScheVal = ScheduleManager::GetCurrentScheduleValue(state, thisExhCtrl.AvailScheduleNum);
 
-        if (FlowRatio != 0.0) {
+        if (FlowRatio >= 0.0) {
```

Comment:
This will fail to zero out the flows if this is called with FlowRatio = 0.0 (which I think is a valid possible case).

Reply:
Oh, whoops >= - sorry this will work.